src/api/app/models/configuration.rb

```diff
@@ -96,7 +96,7 @@ def update_from_options_yml
   # the database or in migrations when there is no backend
   # running
   def delayed_write_to_backend
-    delay.write_to_backend
+    ConfigurationWriteToBackendJob.perform_later(id)
```

Comment:
`Configuration` is still based on a row in the database with an id, and `write_to_backend` is an instance method on the model so I need the id to call it. This implementation isn't any different than how it was before.